bibliopixel/drivers/visualizer.py

```diff
@@ -38,14 +33,15 @@ def __init__(self, num=0, width=0, height=0, pixelSize=15, port=1618, stayTop=Fa
             if "windows" in operating_system:
                 exe_string = "start python"
             elif "darwin" in operating_system:
-                exe_string = "python"
+                exe_string = "python2.7"
                 suffix = "&"
             else:
-                exe_string = "python"
+                exe_string = "python2.7"
```

Comment:
Ewww.... yeah. My main concern was easy cross-platform which is why I used tkinter. Sadly no good Python UI's have cropped up in the last 2 years.